napari/_qt/qt_main_window.py

```diff
@@ -204,6 +215,46 @@ def _add_plugins_menu(self):
         report_plugin_action.triggered.connect(self._show_plugin_err_reporter)
         self.plugins_menu.addAction(report_plugin_action)
 
+    def _show_plugin_list(self):
```

Comment:
Not sure where to put this comment, but the grid text is editable, which I don't think it should be 
![image](https://user-images.githubusercontent.com/6531703/78402935-5640c280-75b0-11ea-983b-f9f99129d572.png)

Reply:
Can you also make the url underlined so it is more clear that it is a link?

Reply:
Also do you see how in the top left there is a bright white vertical line? Can you make that the same as the others? 
![image](https://user-images.githubusercontent.com/6531703/78403074-9bfd8b00-75b0-11ea-8bb8-1116c8c3b95c.png)